irpmonc/irpmonc.cpp

```diff
@@ -145,6 +145,23 @@ static std::wstring _loadServiceName;
 static std::wstring _unloadServiceName;
 
 
+static void _print_sys_errmsg(DWORD err)
+{
+	LPVOID msg;
+
+	if (!FormatMessage(
+	    FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM |
+	    FORMAT_MESSAGE_IGNORE_INSERTS,
+	    NULL, err, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
+	    (LPTSTR) &msg, 0, NULL)) {
+		fprintf(stderr, "Could't get error string for error %ld\n", err);
+		return;
+	}
+	wprintf(L"%s", msg);
```

Comment:
I would suggest using `fwprintf` in order to send the string to `stderr`.

Reply:
That makes sense. Repushed.